index.bs

```diff
@@ -1527,6 +1555,42 @@ example of the latter, when the user is accessing the [=[RP]=] from a given clie
 use a [=roaming authenticator=] which was originally registered with the [=[RP]=] using a different client.
 
 
+### Attestation Conveyance Preference enumeration (enum <dfn enum>AttestationConveyancePreference</dfn>) ### {#attestation}
+
+[=[RPS]=] may use {{AttestationConveyancePreference}} to specify their preference regarding attestation conveyance
+during credential generation.
+
+<pre class="idl">
+    enum AttestationConveyancePreference {
+        "none",
+        "indirect",
+        "direct"
+    };
+</pre>
+
+<div dfn-type="enum-value" dfn-for="AttestationConveyancePreference">
+    *   <dfn>none</dfn> - indicates that the [=[RP]=] is not interested in [=authenticator=] [=attestation=].
+        The client may replace the [=AAGUID=] and [=attestation statement=] generated
+        by the authenticator with meaningless client-generated values. For example, in order to avoid having to obtain 
+        [=user consent=] to relay uniquely identifying information to the [=[RP]=], or to save a roundtrip to a Privacy CA. 
+
+        This is the default value.
+
+    *   <dfn>indirect</dfn> - indicates that the [=[RP]=] prefers an [=attestation=] conveyance yielding to verifiable 
```

Comment:
s/yielding to verifiable/yielding verifiable/

Reply:
doh, thx!